learning/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -0,0 +1,187 @@
+.. _doc_gdscript_styleguide:
+
+GDScript Style Guide
+====================
+
+Description
+-----------
+
+This styleguide lists conventions to write elegant GDScript. The goal is
+to encourage writing clean, readable code and promote consistency across
+projects, discussions, and tutorials. Hopefully, this will also
+encourage development of auto-formatting tools.
+
+Since GDScript is close to Python, this guide is inspired by Python's
+`PEP 8 <https://www.python.org/dev/peps/pep-0008/>`__ programming
+styleguide.
+
+*Note:* Godot's built-in script editor uses a lot of these conventions
```

Comment:
Use the docs' note block
`.. note::`
If you keep it with multiple lines you'll have to create a block with 4 spaces as indent
```
.. note::

    Godot's built-in script editor uses a lot of these conventions
    by default. Let it help you.
```

Reply:
Aside from that I think it's good

Reply:
Good one - I did that in my tutorial, but hadn't gone back and added them to this one